src/cmd/moby/run_qemu.go

```diff
@@ -73,18 +91,38 @@ func runQemu(args []string) {
 	} else {
 		qemuArgs = append(qemuArgs, "-enable-kvm")
 		qemuArgs = append(qemuArgs, "-machine", "q35,accel=kvm:tcg")
+		if container {
+			dockerArgs = append(dockerArgs, "", "")
+			copy(dockerArgs[5:], dockerArgs[3:])
```

Comment:
To add `--device /dev/kvm` if you are running on a system that supports KVM.
The `dockerArgs` set up above ends with image name and thus can't be appended to as `--device` has to come before it.

I could just do a plain old `append` here and then have this afterwards.
```
if container {
    dockerArgs = append(dockerArgs, QemuImg)
}
```